tests/impl.cpp

```diff
@@ -6081,7 +6081,15 @@ result_t test_mm_stream_si32(const SSE2NEONTestImpl &impl, uint32_t iter)
 
 result_t test_mm_stream_si64(const SSE2NEONTestImpl &impl, uint32_t iter)
 {
-    return TEST_UNIMPL;
+    const int64_t a = (const int64_t) impl.mTestInts[iter];
+#if defined(__x86_64__) || defined(__i386__) || defined(__LLP64__)
```

Comment:
`__LLP64__` is a matter of ABI. Can you narrow down? That is, x86_64 with Linux, Windows, macOS, etc.

Reply:
I have added sub-cases for different OSes under `_86_64`